ghost/members-api/test/unit/lib/repositories/member.test.js

```diff
@@ -375,12 +376,65 @@ describe('MemberRepository', function () {
             ]);
         });
 
+        it('subscribes a member to a list of active newsletters', async function () {
```

Comment:
🤔 maybe we can get rid of the previous `subscribes a member to a specified newsletter` test and update this one to be

```suggestion
        it('subscribes a member to the specified newsletters', async function () {
```